CHANGELOG.md

```diff
@@ -1,60 +1,60 @@
 <a name="Pending Release"></a>
-## [Pending Release](https://github.com/lightstep/lightstep-tracer-java/compare/master...0.12.5)
+## [Pending Release](https://github.com/lightstep/lightstep-com.lightstep.tracer-java/compare/master...0.12.5)
```

Comment:
I thought we should maintain the changelog history as we maintained the file history, but maybe it makes sense to start fresh for this repo alone.